frontend/src/component/common/Search/Search.tsx

```diff
@@ -143,6 +145,22 @@ export const Search = ({
             }
         },
     );
+
+    const recordFocusEvent = (e: React.FocusEvent<HTMLInputElement>) => {
+        if (!e.relatedTarget) {
```

Comment:
Cool, what sets the relatedTarget prop? The hotkey hook?

Reply:
good question, it does not work as I expected